assets/section-main-product.css

```diff
@@ -298,6 +298,10 @@ fieldset.product-form__input .form__label {
 
 /* Product info */
 
+.different-color-scheme {
+  padding: 2rem;
+}
```

Comment:
I think on mobile we could remove the spacing on the first child of the product info and add a margin-top to the container maybe: 

![](https://screenshot.click/12-16-sfrqy-aqode.png)

Reply:
I agree! Maybe another 1.5rem